core/api/src/services/notifications/index.ts

```diff
@@ -65,6 +64,9 @@ export const NotificationsService = (): INotificationsService => {
       const response = await notificationsGrpc.getNotificationSettings(
         request,
         notificationsGrpc.notificationsMetadata,
+        {
```

Comment:
:thinking: shouldn't we use something similar to https://github.com/GaloyMoney/galoy/blob/main/core/api/src/services/lnd/index.ts#L801 ?

Reply:
That would work too, but I don't see a problem with relying on grpc.